### DISCUSSION TITLE
Interpolation like Ansible?

### DISCUSSION THREAD

**Discussion by applied-mathematician** [omry/omegaconf#723] [Q&A] [answered]
Is there an easy way to accept string interpolation in the same format as Ansible / Jinja, .i.e. {{ some_var }}?  I would prefer to use a single conf directory for both provisioning my VM and launching my application.  I can get by using full paths for every variable labeling a directory, but interpolation would be more convenient.  Alternatively, I could generate an app.conf file during provisioning, but that would feel a little redundant.

**Comment by Jasha10** [comment, marked as answer]
Hi @applied-mathematician,

If I understand correctly, this is a question about whether OmegaConf is able to parse interpolations that are formatted with two pairs of enclosing braces, e.g. "{{ some_var }}". The answer is unfortunately no, OmegaConf does not currently support this; only the "${some_var}" syntax is supported for variable interpolations.

**Reply by omry** [reply]
Correct.
The interpolation syntax in OmegaConf is very rich and it would take a very compelling argument to justify any changes.
Compatibility with other systems is not a goal. In addition, this will actually be a breaking change for anyone mixing jinja with OmegaConf configs right now, for example [configen](https://github.com/facebookresearch/hydra/blob/master/tools/configen/configen/templates/sample_config.yaml).
